### PR TITLE
Renamed submitContribution to proposeContributionReward

### LINKED ISSUE #204: change the method name `submitContribution` to `submitContributionReward`
Change the method name `submitContribution` to `submitContributionReward`

### PR DESCRIPTION
**Pull request by Alonski**
Fixes #204

### CODE REVIEW

File: test/contributionreward.js [hunk 1]
```diff
@@ -72,10 +72,10 @@ contract('ContributionReward', function(accounts) {
      assert.equal(await testSetup.contributionReward.isRegistered(testSetup.org.avatar.address),true);
     });
 
-    it("submitContribution log", async function() {
+    it("proposeContributionReward log", async function() {
       var testSetup = await setup(accounts,0,0);
       // await testSetup.contributionReward.registerOrganization(testSetup.org.avatar.address);
```
Comment (orenyodfat) [inline test/contributionreward.js]: Though this PR did not update that ..
I would advice to remove this commented code.

File: test/contributionreward.js [hunk 2]
```diff
@@ -84,15 +84,15 @@ contract('ContributionReward', function(accounts) {
       assert.equal(tx.logs[0].event, "LogNewContributionProposal");
      });
 
-     it("submitContribution fees", async function() {
+     it("proposeContributionReward fees", async function() {
        var testSetup = await setup(accounts,14,10);
 
        // await testSetup.contributionReward.registerOrganization(testSetup.org.avatar.address);
```
Comment (orenyodfat) [inline test/contributionreward.js]: Though this PR did not update that ..
I suggest  to remove this commented code.

File: test/contributionreward.js [hunk 3]
```diff
@@ -122,10 +122,10 @@ contract('ContributionReward', function(accounts) {
        }
       });
 
-      it("submitContribution check owner vote", async function() {
+      it("proposeContributionReward check owner vote", async function() {
         var testSetup = await setup(accounts);
         // await testSetup.contributionReward.registerOrganization(testSetup.org.avatar.address);
```
Comment (orenyodfat) [inline test/contributionreward.js]: Though this PR did not update that ..
I suggest to remove this commented code.

**Comment by Alonski** [comment]
I removed the commented code that you asked for...
I tried rebasing and somehow it still shows two comments.
Need to figure out rebasing 😅

**Pull request merged**